CopilotKit/packages/backend/src/lib/openai-adapter.ts

```diff
@@ -49,18 +60,36 @@ export class OpenAIAdapter implements CopilotKitServiceAdapter {
         }
       });
 
-      const stream = this.openai.beta.chat.completions.stream({
-        model: this.model,
-        ...forwardedProps,
-        stream: true,
-        messages: messages as any,
-      });
+      let stream: ChatCompletionStream;
+
+      try {
```

Comment:
This was previously not caught and resulted in an unhandled rejection

Reply:
nice catch